server/src/db/serializers/DogParksSerializer.js

```diff
@@ -11,8 +13,8 @@ class DogParksSerializer {
         return newParks
     }
 
-    static detailsForShow(dogPark) {
-        const allowedAttributes = ["id", "name", "address", "description", "updatedAt"]
+    static async detailsForShow(dogPark) {
+        const allowedAttributes = ["id", "name", "address", "description", "updatedAt", "reviews"]
```

Comment:
at this point, `reviews` has not yet been queried or assigned as a property for the `dorPark` being serialized (not until lines 41-43). When serializing the object we want to the `attributes` should ideally be properties from that table, then associations are handled separately from the main object